sdcm/utils/common.py

```diff
@@ -1284,17 +1284,21 @@ def wait_ami_available(client, ami_id):
                 )
 
 
-def update_certificates():
+def update_certificates(db_csr='data_dir/ssl_conf/example/db.csr', cadb_pem='data_dir/ssl_conf/cadb.pem',
```

Comment:
Does this function create a new certificate every time it runs?

Reply:
yes, every time it runs, the openssl command creates a new certificate.

Reply:
what's the point of introducing those new parameters is they are not used anywhere ?

Reply:
In the beginning i used them, but at the end, it wasn't fully necessary for my case. Anyway, I think it is good we can control the input and output files locations and names.